modules/permutiveRtdProvider.js

```diff
@@ -260,6 +268,39 @@ function readSegments (key) {
   }
 }
 
+/**
+ * Bidder-specific extension data for use in the ORTB2 object.
+ */
+const extensionDataByBidder = {
+  ix: {
+    segtax: '4'
+  }
+}
+
+const unknownIabSegmentId = '_unknown_'
+
+/**
+ * Bidder-specific functions to apply to an array of segment IDs.
+ * Each function should return an a new array of transformed segment IDs.
```

Comment:
Nit: this is a map and a filter which I wouldn't nec. got from the description

Reply:
My intention for this comment wasn't to specify how each function should generate a new array of segment IDs, only that it should return one – make sense?

Reply:
I've updated the comment now that the way transformations work has changed, hopefully it's clearer now.